samples/hello/Makefile

```diff
@@ -1,5 +1,6 @@
 XBE_TITLE = nxdk\ sample\ -\ hello
 GEN_XISO = $(XBE_TITLE).iso
 SRCS = $(CURDIR)/main.c
-NXDK_DIR = $(CURDIR)/../..
+NXDK_DIR ?= $(CURDIR)/../..
+CFLAGS = -fstack-protector-all
```

Comment:
Was this line supposed to be added here?

Reply:
Nope, left that in accidentally. It's fixed now.